doc/whats_new/v0.21.rst

```diff
@@ -146,6 +146,10 @@ Support for Python 3.4 and below has been officially dropped.
   float64 for the ``stratified`` strategy. :issue:`13266` by
   :user:`Christos Aridas<chkoar>`.
 
+- |Fix| Fixed a bug in :class:`dummy.DummyClassifier` where 1d dimensional y
```

Comment:
"1d dimensional y with ndim=2" makes no sense to me. Do you mean "column vector"?

Reply:
Yep, it was a copy paste from the old issue, sorry. Fixed.